src/centralized_telescope/round.rs

```diff
@@ -1,5 +1,7 @@
 //! ALBA's Round structure and associated functions
 
+#![doc = include_str!("../../docs/centralized/round.md")]
```

Comment:
So we basically include this page in two different places. Perhaps we can include it only once and add a link at the other place? Otherwise, the structure becomes a bit confusing.

Reply:
I think it's better to decide when we have the content. I keep it as is then we can discuss it when I work on related PR.
WDYT?